bin/pygrb/pycbc_pygrb_pp_workflow

```diff
@@ -139,25 +183,25 @@ logging.info("Created log file %s", wf_log_file.storage_path)
 
 out_dir = rdir.base
 _workflow.makedir(out_dir)
-files = _workflow.FileList([])
 
 #
-# Create information table about the GRB trigger and plot the search grid
+# Opening page
 #
-info_table_node, grb_info_table = _workflow.make_info_table(wflow, 'pygrb_grb_info_table', out_dir)
+# Create the information table about the GRB trigger
+info_table_node, grb_info_table = \
+    _workflow.make_pygrb_info_table(wflow, 'pygrb_grb_info_table', out_dir)
```

Comment:
Run this one through `black` to get correct formatting .... At some point we'll figure out how to automate this.

Reply:
(Actually, might make sense to do a pass over this code with `black` *after* you've merged in the patches you're working on, and have that as a separate patch).

Reply:
I was going to suggest to keep this for the very end, yes.